jujugui/static/gui/src/app/components/profile/model-list/model-list.js

```diff
@@ -74,6 +74,29 @@ class ProfileModelList extends React.Component {
     });
   }
 
+  _confirmDestroy(modelUUID) {
+    this.setState({notification: null});
+    this.props.destroyModel(modelUUID, (error, data) => {
+      if (error) {
+        this.props.addNotification({
+          title: 'Error destroying model',
+          message: error,
+          level: 'error'
+        });
```

Comment:
`return;` after this msg else you could give two notifications for the same error....or maybe not....see comment below about multiple error locations.